webrtc.html

```diff
@@ -4946,6 +4947,23 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
               <code>degradationPreference</code> indicates which is
               preferred.</p>
             </dd>
+            <dt><dfn><code>upscaleAllowed</code></dfn> of type
+            <span class="idlMemberType"><a>boolean</a></span>,
+            defaulting to <code>false</code></dt>
+           <dd>
+             <p>When, and only when, this value is set true the browser MAY
+            upscale video temporally or spatially, or upscale the audio sampling
+            rate, or upscale the number of bits in the audio sample that this
+            <code><a>RtpSender</a></code> sends over RTP.
```

Comment:
I don't remember that conversation about audio sample rate.  Can you point me to minutes about it? Maybe I just zoned out during that time.

If we do have consensus to add that as well, I think something like this would be more clear:

ResolutionScalingPolicy resolutionScalingPolicy = "down-only";  // or "up-and-down"
SampleRatePolicy sampleRatePolicy = "change-allowed";  // "maintain-rate"